src/Elastic.Markdown/IO/MarkdownFile.cs

```diff
@@ -43,11 +43,23 @@ public DocumentationGroup? Parent
 	public string? UrlPathPrefix { get; }
 	private MarkdownParser MarkdownParser { get; }
 	public YamlFrontMatter? YamlFrontMatter { get; private set; }
+
+	public string? TitleRaw
+	{
+		get => _titleRaw;
+		private set
+		{
+			Title = value?.StripMarkdown();
```

Comment:
I meant it reverse e.g:

Title is the sanitized version. 

```csharp

public string TitleRaw { get; private set; }

public string? Title
{
	get => _titleRaw;
	private set
	{
        Title = value?.StripMarkdown();
		TitleRaw = value;
    }
}
```

That way the change is not that pervasive, the only place we need access to the markdown is in the rendering slice.

Reply:
Aha!

TitleRaw sounded to me like "Raw Markdown String" 😅

Reply:
OK. I think I got it right now.

https://github.com/elastic/docs-builder/pull/290/commits/e00d555e7d3d962ca93f06798074c24b261c20fe